xtokens/src/lib.rs

```diff
@@ -89,7 +89,7 @@ pub mod module {
 		type SelfLocation: Get<MultiLocation>;
 
 		/// Minimum xcm execution fee paid on destination chain.
-		type MinXcmFee: GetByKey<MultiLocation, u128>;
+		type MinXcmFee: GetByKey<MultiLocation, Option<u128>>;
```

Comment:
can we implements a default disabled struct which do not support it, then runtime can just use `DisabledParachainFee`?
```rust
pub struct DisabledParachainFee;
impl GetByKey<MultiLocation, Option<u128>> for DisabledParachainFee {
    fn get(k: &MultiLocation) -> Option<u128> {
        None
    }
}
```

Reply:
Yeah sounds good.

Reply:
I was thinking to put it in either traits or xcm-support, since xtokens itself does not have any other default impl. 
Chose traits so you don't have to import another crate just for this.